book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,184 @@
+You will need to use a terminal emulator to follow.
 
-1. Install miniconda by following the instructions for your Operating System (Windows, MacOS, Linux) at this link: https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+You will also need to install Python3.
+You can check which specific version of Python3 the build uses in [netlify.toml](https://github.com/the-turing-way/the-turing-way/blob/main/netlify.toml).
+However, versions close to that version are also likely to work.
 
-2. Open your terminal app and run the `conda init` command in your terminal. You should see `(base)` in your prompt indicating that conda was successfully installed and you are now in its base environment.
-    - Note that if you are using Windows, you will need to open a program called `Anaconda prompt` instead of using `cmd`.
+Other command line tools you will need are,
 
-3. Create a new environment and install a modern version of Python into it:
+- `git`
+- `make`
 
-   ```console
-   conda create --name the-turing-way python=3.13
-   ```
+## Step-by-step Guide
 
-4. Activate the environment with:
+### Clone The Repository
 
-   ```console
-   conda activate the-turing-way
-   ```
+```console
+git clone https://github.com/the-turing-way/the-turing-way
+```
 
-   Any commands we run with Python or pip from now on will use the versions of Python and pip installed into _this_ conda env, not any others.
+````{note}
+The repository is quite large and cloning may take a long time on slower internet connections.
+You can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#).
+Specifically, focusing on blobless clones, involves utilizing the `--filter=blob:none` option in the git clone command.
 
-5. Clone _The Turing Way_ repository from GitHub to your machine using the command: 
+By using `--filter=blob:none`, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a git checkout.
 
-   ```console
-   git clone https://github.com/the-turing-way/the-turing-way
-   ```
+Here's the command to create a blobless clone of the book:
 
-   ````{note}
-   To address potential barriers with slow internet connections due to the large size of this repository, you can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#). Specifically, focusing on blobless clones, which are efficient for developers, involves utilizing the `--filter=blob:none` option in the git clone command.
+```console
+git clone --filter=blob:none https://github.com/the-turing-way/the-turing-way.git
+```
+````
 
-   By using `--filter=blob:none`, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a git checkout
+### Create a Virtual Environment
```

Comment:
Does this section work on windows?

Reply:
Presumably, do you have a Windows machine to test it on?

Reply:
No, I don't have one. But this section is targeting those who doesn't have working developer environments set up, and they may more likely be on windows, too, thus it should work.